pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "Cython", "pyfftw", "h5py", "scipy"]
```

Comment:
Only packages used in the build script should be included here. `pyfftw` will trigger a pip install and fail on M1 mac.